core/src/test/kotlin/org/evomaster/core/problem/graphql/GraphQLUtilsTest.kt

```diff
@@ -0,0 +1,382 @@
+package org.evomaster.core.problem.graphql
+
+import com.google.gson.Gson
+import org.evomaster.core.problem.graphql.schema.SchemaObj
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.util.*
+
+
+class GraphQLUtilsTest {
+
+    val unionTag = "#UNION#"
+    val interfaceTag = "#INTERFACE#"
+
+    @Test
+    fun graphTest() {
+
+        val json = PetClinicCheckMain::class.java.getResource("/graphql/MelodyRepo2.json").readText()
+
+        val state = GraphQLActionBuilder.TempState()
+        val gson = Gson()
+        val schemaObj: SchemaObj = gson.fromJson(json, SchemaObj::class.java)
+        GraphQLActionBuilder.initTablesInfo(schemaObj, state)
+        val queryGraph: MutableMap<String, GraphQLUtils.GraphInfo> = mutableMapOf()
+        GraphQLUtils.constructGraph(state, "query", " ", queryGraph, mutableListOf(), mutableSetOf())
+        Assertions.assertEquals(4, GraphQLUtils.getGraphSize(queryGraph))
```

Comment:
you can do a `*` static import, so can write `assertEquals` directly instead of repeating `Assertions.` all the times

Reply:
Done